homeassistant/components/generic/config_flow.py

```diff
@@ -314,9 +330,21 @@ async def async_step_user(
                         # The automatically generated still image that stream generates
                         # is always jpeg
                         user_input[CONF_CONTENT_TYPE] = "image/jpeg"
-
-                    return self.async_create_entry(
-                        title=name, data={}, options=user_input
+                    self.cached_user_input = user_input
```

Comment:
Optionally consider renaming the member variable to `user_input` and `title`. My thought is that the preview storage feels more like a cache, but this is the actual source of truth copy of the user input stored for persistence in the config entry, so it could help disambiguate.

Reply:
Done 06f456e7f